pytest_fixtures/core/contenthosts.py

```diff
@@ -292,6 +292,7 @@ def bootc_host():
     with Broker(
         workflow='deploy-bootc',
         host_class=ContentHost,
+        target_template='tpl-bootc-rhel-10.0',
         deploy_network_type='ipv6' if settings.server.is_ipv6 else 'ipv4',
     ) as host:
```

Comment:
conf/content_host.yaml:
```yaml
content_host:
  rhel10_bootc:  # here I am not sure if it is the best name
    vm:
      workflow: deploy-bootc
      target_template: tpl-bootc-rhel-10.0 
```
```python
@pytest.fixture(params=[{'rhel_version': '10_bootc'])  # this could eventually go to config as well if we intent to test only a single (bootc) RHEL version.
def bootc_host():
    """Fixture to check out boot-c host"""
```


```suggestion
    with Broker(
        **host_conf(request),
        host_class=ContentHost,
        deploy_network_type='ipv6' if settings.server.is_ipv6 else 'ipv4',
    ) as host:
```

And the tests will then need to be marked with
```python
@pytest.mark.no_containers
```

Seems like a lot of hassle, but I find it somewhat more maintainable than having variables hardcoded in the code.

I would consider this comment to be non-blocking.

Reply:
We are going to be doing more work with Bootc in 6.18 - I'll take this good feedback and apply that during that work, so that this PR can get closed in a timely fashion.